pom.xml

```diff
@@ -503,37 +508,6 @@
                     <version>1.3</version>
                 </plugin>
                 <plugin>
-                    <groupId>org.bsc.maven</groupId>
```

Comment:
> So if you think that's the lesser evil, let's do it in order to move forward.

Make it so! ( I added another comment to be more explicit) 

For now I think it is the lesser evil indeed.